.github/workflows/cron-names.yml

```
@@ -0,0 +1,52 @@
+name: Cron migrate IPNS records to w3name
+
+on:
+  schedule:
+    - cron: '* */24 * * *'
+  workflow_dispatch:
+    inputs:
+      env:
+        required: true
+        description: The env to run the cron against. Default is staging.
+        options: 
+          - staging
+          - production
+        default: staging
+
+jobs:
+  update:
+    name: Export IPNS records to w3name
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        env: ['staging', 'production']
+    timeout-minutes: 60
+    steps:
+      - uses: actions/checkout@v2
+        with: 
+          fetch-depth: 0
+      - name: Checkout latest cron release tag
+        run: |
+          LATEST_TAG=$(git describe --tags --abbrev=0 --match='cron-*')
+          git checkout $LATEST_TAG
+      - name: Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16
+
+      - name: Install dependencies
+        uses: bahmutov/npm-install@v1
+
+      - name: Run job
+        env:
+          DEBUG: '*'
+          ENV: ${{ matrix.env }}
```

From looking at this is done in other configs, I think that maybe `matrix.env` will only get the value from the `matrix` setting, not from the `inputs` section in the `workflow_dispatch`.

```suggestion
          ENV: ${{ matrix.env || github.event.inputs.env }}
```

This is only a guess though. [Stack Over doesn't seem to know!](https://stackoverflow.com/questions/69781005/combine-dynamic-github-workflow-matrix-with-input-values-and-predefined-values)